index.html

```diff
@@ -0,0 +1,115 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.4.0/css/font-awesome.css" />
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@700;900&display=swap" rel="stylesheet">
+    <link href="https://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/dcdd89c74c.js"></script>
+    <script src="https://kit.fontawesome.com/8d20eebc7a.js"></script>
+    <script src="mobile.js" defer></script>
+    <script src="prog.js" defer></script>
+    <script src="prog-spe.js" defer></script>
+    <title>Capstone </title>
+</head>
+<body>
+    <header>
+        <div class="login-details">
+            <ul class="login">
+                <li class="login-link"><a href="#"><i class="fa-brands fa-facebook fa-1.5x"></i></a></li>
+                <li class="login-link"><a href="#"><i class="fa-brands fa-twitter fa-1.5x"></i></a></li>
+                <li class="login-link"><a href="#">English</a></li>
+                <li class="login-link"><a href="#">My page</a></li>
+                <li class="login-link"><a href="#">og out</a></li>
+            </ul>
+        </div>
+        <nav class="navbar">
+            <div class="logo">
+                <img src="project-image/logo.jpg" alt="image">
+            </div>
+            <ul class="desktop-menu">
+                <li class="nav-item">
+                    <a href="about.html" class="nav-link active">About</a>
+                </li>
+                <li class="nav-item">
+                    <a href="#program" class="nav-link">Services</a>
+                </li>
+                <li class="nav-item">
+                    <a href="#join" class="nav-link">Instructors</a>
+                </li>
+                <li class="nav-item">
+                    <a href="#sponsor" class="nav-link">Sponsor</a>
+                </li>
+                <li class="nav-item">
+                    <a href="#news" class="nav-link">News</a>
+                </li>
+                <li class="nav-item">
+                    <p class="campaign">Apply Programs</p>
+                </li>
+            </ul>
+            
+            <div class="mobile-menu">
+                <span class="bar"></span>
+                <span class="bar"></span>
+                <span class="bar"></span>
+            </div>
+        </nav>
+    </header>
```

Comment:
- Kindly arrange the logo, nave menu, and top manu to appropriate place as the given design to have better UI 👍 

#### Your Design 👇
<img width="957" alt="2023-08-10_12-00" src="https://github.com/abel12-pro/Capstone-1/assets/52010789/6c1dee9e-320e-48d8-b9a8-ef5034a3bdb7">

#### Given Design 👇
<img width="673" alt="2023-08-10_11-57" src="https://github.com/abel12-pro/Capstone-1/assets/52010789/e6d97857-d65c-405f-bbaf-e98ba4768e1e">